### PR TITLE
Prevent Metadata Time System Error for Missing Objects

### LINKED ISSUE #7476: Erroneous "Telemetry metadata does not match the active time system" notifications
#### Summary
Open MCT shows the above error incorrectly when an object is missing. This notification is intended to inform the user when a misconfiguration has occurred.

An attempt was made to fix this issue previously but it had either regressed or the issue was never completely solved - https://github.com/nasa/openmct/issues/6349#issuecomment-1462649780

#### Expected vs Current Behavior
The Telemetry Metadata error should only be shown for _extant_ objects, and not missing objects.

#### Steps to Reproduce

1. Navigate to the "TAB - DRIVING RAILS" display in the VIPER Open MCT test environment
2. Observe a notification that states "Telemetry metadata does not match the active time system"

#### Environment

* Open MCT Version: 4.0.x 0aceb4b590be32d0abfc172b972b82b8c2bbcbfd
* Deployment Type: VIPER Test

#### Impact Check List

- [x] Data loss or misrepresented data?
- [ ] Regression? Did this used to work or has it always been broken?
- [x] Is there a workaround available?
- [x] Does this impact a critical component?
- [ ] Is this just a visual bug with no functional impact?
- [ ] Does this block the execution of e2e tests?
- [ ] Does this have an impact on Performance?

### PR DESCRIPTION
**Pull request by jvigliotta**
Closes #7476

### Describe your changes:

Modified Stacked Plots to not show Missing Objects. Added a check in Telemetry Collections for missing objects before displaying telemetry metadata time system error.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this a [notable change](../docs/src/process/release.md) that will require a special callout in the release notes? For example, will this break compatibility with existing APIs or projects that consume these plugins?

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Has this been smoke tested?
* [x] Have you associated this PR with a `type:` label? Note: this is not necessarily the same as the original issue.
* [x] Have you associated a milestone with this PR? Note: leave blank if unsure.
* [ ] Is this a breaking change to be called out in the release notes?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [x] Changes appear to address issue?
* [x] Reviewer has tested changes by following the provided instructions?
* [x] Changes appear not to be breaking changes?
* [x] Appropriate automated tests included?
* [x] Code style and in-line documentation are appropriate?

### CODE REVIEW

File: .webpack/webpack.prod.js [hunk 1]
```diff
@@ -15,5 +15,5 @@ export default merge(common, {
       __OPENMCT_ROOT_RELATIVE__: '""'
     })
   ],
-  devtool: 'source-map'
+  devtool: 'eval-source-map'
```
[the remark this answers is not in the document]
Comment (jvigliotta) [reply .webpack/webpack.prod.js:18]: oh dang, didn't push that removal

[the remark this answers is not in the document]
Comment (jvigliotta) [reply .webpack/webpack.prod.js]: should be good now

**Review by unlikelyzero — APPROVED** [review]

**Review by ozyx — CHANGES_REQUESTED** [review]
Blocking for now will elaborate later but I have a good reason I swear

File: src/plugins/plot/stackedPlot/StackedPlot.vue [hunk 2]
```diff
@@ -219,6 +219,10 @@ export default {
     },
 
     addChild(child) {
+      if (!this.openmct.objects.isMissing(child)) {
```
[the remark this answers is not in the document]
Comment (jvigliotta) [reply src/plugins/plot/stackedPlot/StackedPlot.vue:222]: Haha, it's nuts this got through, not sure how I missed this in the smoke test, nice catch!

File: src/plugins/plot/stackedPlot/StackedPlotItem.vue [hunk 3]
```diff
@@ -172,7 +172,7 @@ export default {
 
       if (this.openmct.telemetry.isTelemetryObject(plotObject)) {
         this.subscribeToStaleness(plotObject);
-      } else {
+      } else if (!this.openmct.objects.isMissing(plotObject)) {
```
[the remark this answers is not in the document]
Comment (jvigliotta) [reply src/plugins/plot/stackedPlot/StackedPlotItem.vue:175]: Yep, I like that, will do.

**Review by ozyx — APPROVED** [review]
LGTM

**Pull request merged**